src/test/java/com/waverleysoftware/AccessCodeTests.java

```diff
@@ -15,16 +15,19 @@
     @Test(dataProvider = "getPanelData", dataProviderClass = DataSuppliers.class)
     public void accessCodeShouldBeAdded(final Panel... panels) {
         final String conditionForType = "home";
-        getSoftAssert().assertThat(panels[0].getType()).as("Panel Name").isEqualTo(conditionForType);
+        getSoftAssert().assertThat(panels[0].getType())
```

Comment:
as you're using only first element in all test, it'd make sense to pass only single `Panel` instance...